lib/must/must.go

```diff
@@ -0,0 +1,9 @@
+package must
+
+func One[R any](r R, err error) R {
```

Comment:
Must is a fairly common name for a helper which will panic in case of an error in Go.

One is for one return value, because in go you can't overload functions..